crates/starknet/src/starknet/estimations.rs

```diff
@@ -62,16 +66,24 @@ pub fn estimate_message_fee(
         blockifier::transaction::transaction_execution::Transaction::L1HandlerTransaction(
             l1_transaction,
         ),
+        None,
+        None,
     )
 }
 
 fn estimate_transaction_fee(
     state: &mut StarknetState,
     block_context: &blockifier::block_context::BlockContext,
     transaction: blockifier::transaction::transaction_execution::Transaction,
+    charge_fee: Option<bool>,
```

Comment:
why not be just bool instead of Option.

Reply:
To have defaulting (unwrap_or). Since both `estimate_fee` and `estimate_message_fee` require constant values (charge_fee=false, validate=true), I would either need to hardcode the constants in both places, or define some global constants and use them in both places (both estimate and estimate_message).

If estimate and estimate_message and really independent, then they deserve to have the constants specified separately, but I at the time of writing, I thought it would be ok to have the same default values for both.